translate/src/test/java/com/google/cloud/translate/samples/TranslateTextTest.java

```diff
@@ -0,0 +1,121 @@
+        "Georgian", "German", "Greek", "Gujarati", "Haitian", "Hausa", "Hawaiian", "Hebrew",
+        "Hindi", "Hmong", "Hungarian", "Icelandic", "Igbo", "Indonesian", "Irish", "Italian",
+        "Japanese", "Javanese", "Kannada", "Kazakh", "Khmer", "Korean", "Kurdish", "Kyrgyz",
+        "Lao", "Latin", "Latvian", "Lithuanian", "Luxembourgish", "Macedonian", "Malagasy",
+        "Malay", "Malayalam", "Maltese", "Maori", "Marathi", "Mongolian", "Myanmar", "Nepali",
+        "Norwegian", "Pashto", "Persian", "Polish", "Portuguese", "Punjabi", "Romanian",
+        "Russian", "Samoan", "Scots", "Serbian", "Sesotho", "Shona", "Sindhi", "Sinhala",
+        "Slovak", "Slovenian", "Somali", "Spanish", "Sundanese", "Swahili", "Swedish",
+        "Tajik", "Tamil", "Telugu", "Thai", "Turkish", "Ukrainian", "Urdu", "Uzbek",
+        "Vietnamese", "Welsh", "Xhosa", "Yiddish", "Yoruba", "Zulu");
+
+    // Arrange
+    ByteArrayOutputStream bout = new ByteArrayOutputStream();
+    PrintStream out = new PrintStream(bout);
+
+    // Act
+    TranslateText.displaySupportedLanguages(out);
+
+    // Assert
+    String got = bout.toString();
+    for (String language : languages) {
+      assertThat(got).contains(language);
+    }
+  }
+
+  @Test public void testEnglishLangDetection() throws Exception {
+    // Arrange
+    ByteArrayOutputStream bout = new ByteArrayOutputStream();
+    PrintStream out = new PrintStream(bout);
+
+    // Act
+    TranslateText.detectLanguage("With power comes great responsibility.", out);
+
+    // Assert
+    String got = bout.toString();
+    assertThat(got).contains("language=en, confidence=0.79742646");
+  }
+
+  @Test public void testGermanLangDetection() throws Exception {
+    // Arrange
+    ByteArrayOutputStream bout = new ByteArrayOutputStream();
+    PrintStream out = new PrintStream(bout);
+
+    // Act
+    TranslateText.detectLanguage("Mit Macht kommt große Verantwortung.", out);
+
+    // Assert
+    String got = bout.toString();
+    assertThat(got).contains("language=de, confidence=0.9293963");
+
+  }
+
+  @Test public void testDefaultIdentityTranslation() throws Exception {
+    // Arrange
+    ByteArrayOutputStream bout = new ByteArrayOutputStream();
+    PrintStream out = new PrintStream(bout);
+
+    // Act
+    String proverb = "What you do not wish for yourself, do not do to others.";
+    TranslateText.translateText(proverb, out);
+
+    // Assert
+    String got = bout.toString();
+    assertThat(got).contains(proverb);
+  }
+
+  @Test public void testGermanToSpanishTranslation() throws Exception {
+    // Arrange
+    ByteArrayOutputStream bout = new ByteArrayOutputStream();
+    PrintStream out = new PrintStream(bout);
+
+    // Act
+    TranslateText.translateTextWithOptions("Mit Macht kommt große Verantwortung.", "de", "es", out);
+
+    // Assert
+    String got = bout.toString();
+    assertThat(got).contains("Con el poder viene una gran responsabilidad.");
```

Comment:
Currently, we only get one translation. If this changes it will be good to know why and make modifications to the test appropriately. But, better to know than qualify the test even if accidentally a bad translation is returns which contains  responsabilidad as only relevant word.